### ISSUE TITLE
Appbar should be first in document order

### ISSUE THREAD

**Issue by sdellis** [ProjectMirador/mirador#2023]
Currently, one must tab through the entire app before getting to the Appbar (left-hand vertical workspace menu). Put this before the workspace in the document for a better keyboard-only experience.
